tests/cases/compiler/warnExperimentalBigIntLiteral.ts

```diff
@@ -0,0 +1,6 @@
+// @target: esnext
+
+const normalNumber = 123; // should not error
+let bigintType: bigint; // should not error
+let bigintLiteralType: 123n; // should error when used as type
```

Comment:
when using big int literal as type (eg 123n in `let bigintLiteralType: 123n` ) it should error. Please fix that

Reply:
Do you mean that it should *not* error? If so, what is the recommended way of checking whether a literal is used as a type or a value? Can you use something like `isTypeNode()`?

Reply:
You can check the `.parent`. If it's a literal type node, don't error. If it's a prefix unary expression, you can check that node's parent.